InternetHospitalWebUI/src/app/Components/Authorization/sign-in/sign-in.component.scss

```diff
@@ -1,42 +1,52 @@
 .example-container {
-    display: flex;
```

Comment:
you've changed 47 files in this pr. Why do you call it other minor changes?

Reply:
It`s my fault, i hope it would never happen again